nxdrive/options.py

```diff
@@ -116,6 +116,20 @@ def _is_system_wide() -> bool:
     )
 
 
+class CallableFeatureHandler:
+    """Retrun a callable class"""
```

Comment:
Could you expand the dcostring to explain why it is necessary and how it works (with code example as done at the top of that file)?

- Add type annotations everywhere + docstring in methods.